addons/koji/common/src/main/java/org/commonjava/indy/koji/content/KojiMavenMetadataProvider.java

```diff
@@ -194,7 +194,17 @@ public Metadata getMetadata( StoreKey targetKey, String path )
                                 logger.debug( "Skipping non-POM: {}", archive.getFilename() );
                                 continue;
                             }
-                            SingleVersion singleVersion = VersionUtils.createSingleVersion( archive.getVersion() );
+                            SingleVersion singleVersion = null;
+                            try
+                            {
+                                singleVersion = VersionUtils.createSingleVersion( archive.getVersion() );
+                            }
+                            catch ( InvalidVersionSpecificationException ivse )
```

Comment:
Starting to wish I hadn't made this an unchecked exception...but it would affect a LOT of code to change it.